admin/admin-service/src/main/java/io/mosip/admin/packetstatusupdater/util/EventEnum.java

```diff
@@ -171,7 +171,7 @@ public String getApplicationName() {
 
 	public static EventEnum getEventEnumBasedOnPAcketStatus(PacketStatusUpdateDto ps)
 	{
-		
+		if(null!=ps.getTransactionTypeCode()) {
```

Comment:
Reverse the condition and reduce the if statement. Basically check for null and just return immediately rather than creating a big block.